### DISCUSSION TITLE
send token or send btc example

### DISCUSSION THREAD

**Discussion by xll970211** [1200wd/bitcoinlib#359] [Q&A] [answered]
If I have a privatekey, this account have btc coin, I want send btc to another account, what should I do? Please give me a example, I want to send token to another account，give me a example，Thank you very much

**Comment by mccwdev** [comment, marked as answer]
Easiest is to create a wallet and then create a transaction:

``` python
pk = 'tprv8ZgxMBicQKsPeTWfDcHDjngYVa8LR4VQnn4GtoJLXEtQYpRRkRt6TsqKp2nBRfnJt9vjUTREV8YpjPr8jecxNh1USymackFnAHnD7MEQo7r'
w = wallet_create_or_open('mycoins', pk)
w.scan(scan_gap_limit=1)
w.info()
w.send_to('n1UDMjicDJ6c2Kuv8eHLcYrZUmGGXfH2Nu', 10000)
```

or if you have just a single key with unspent outputs:

``` python
pk = 'tprv8ZgxMBicQKsPeTWfDcHDjngYVa8LR4VQnn4GtoJLXEtQYpRRkRt6TsqKp2nBRfnJt9vjUTREV8YpjPr8jecxNh1USymackFnAHnD7MEQo7r'
w = wallet_create_or_open('mycoins_single', pk, scheme='single')
w.transactions_update()
w.info()
w.send_to('n1UDMjicDJ6c2Kuv8eHLcYrZUmGGXfH2Nu', 10000)
```

**Reply by xll970211** [reply]
Thank you. I successfully sent the transaction, but the transaction can't find, and when I used the wallet_create_or_open method and provided the key returned from creating the account earlier, the address of the account used for sending the transaction changed.
